<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>501</width>
      <height>381</height>
     </rect>
    </property>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>register</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="userLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="passlabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="userLabel_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
